templates/notifyPopup.hbs

```diff
@@ -1,7 +1,7 @@
 {{! make the _globals object in course.json available to this template}}
 {{import_globals}}
 
-<div {{#each _attributes}}{{@key}}="{{this}}" {{/each}} class="notify__popup notify__popup-type-{{_type}}{{#if _classes}} {{_classes}}{{/if}}">
+<div {{#each _attributes}}{{@key}}="{{this}}" {{/each}} class="notify__popup notify__popup-type-{{_type}}{{#if _classes}} {{_classes}}{{/if}}{{#if _graphic._src}} has-graphic{{/if}}{{#if _imageAlignment}} is-graphic-{{_imageAlignment}}{{/if}}">
```

Comment:
Minor point, but for consistency with similar functionality ([Hotgraphic image alignment](https://github.com/adaptlearning/adapt-contrib-hotgraphic/blob/a34c6b7973d8015a588faae080c9baaf0b438ec6/less/hotgraphicPopup.less#L87)) should we keep the same naming convention for selectors e.g. `.align-image-left` instead of `.is-graphic-left`?